### PR TITLE
"Exit Review Mode" is confusing

### LINKED ISSUE #3637: "Exit Review Mode" is confusing
Full discussion is in https://github.com/microsoft/vscode-pull-request-github/issues/1048

"Exit Review Mode" makes it sound like all the reviewing feature (comments, views, etc.) will be removed. In actuality, all we're doing is checking out the default branch for the repository. 

Ideally, this action should have a label that indicates that the default branch will be checked out using standard git terms _and_ that this is the way to stop reviewing the current PR. Some options discussed:
- "Leave PR Branch"
- "Back to 'main'"
- "Checkout 'main'"
None of these quite accomplish the above however. For now, we'll try "Checkout 'main'" and see how that goes.

### PR DESCRIPTION
**Pull request by alexr00**
Fixes #3637

### CODE REVIEW

**Review by aeschli — APPROVED** [review]

**Review by XavierMP14 — COMMENTED** [review]
View

**Review by XavierMP14 — COMMENTED** [review]
Pull repuesto

File: webviews/activityBarView/exit.tsx [hunk 1]
```diff
@@ -8,16 +8,16 @@ import { GithubItemStateEnum } from '../../src/github/interface';
...
 
-const ExitButton = ({ isBusy, onClick }: { isBusy: boolean, onClick: () => Promise<void> }) => {
+const ExitButton = ({ repositoryDefaultBranch, isBusy, onClick }: { repositoryDefaultBranch: string, isBusy: boolean, onClick: () => Promise<void> }) => {
 	return (<button title="Switch to a different branch than this pull request branch" disabled={isBusy} onClick={onClick}>
-		Exit Review Mode
+		Checkout '{repositoryDefaultBranch}'
 	</button>);
 };
 
-const ExitLink = ({ onClick }: { onClick: () => Promise<void> }) => {
```
Comment (XavierMP14) [inline webviews/activityBarView/exit.tsx:17]: F

File: webviews/common/cache.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export interface PullRequest {
 	assignees: IAccount[];
 	commitsCount: number;
 	milestone: IMilestone;
-	repositoryDefaultBranch: any;
+	repositoryDefaultBranch: string;
 	/**
 	 * User can edit PR title and description (author or user with push access)
```
Comment (XavierMP14) [inline webviews/common/cache.ts:48]: View

**Review by XavierMP14 — COMMENTED** [review]
X

**Pull request merged**